tests/functional/base_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	k8s_errors "k8s.io/apimachinery/pkg/api/errors"
 )
 
-func DefaultHorizonTemplate(name types.NamespacedName) *horizon.Horizon {
+func DefaultHorizonTemplate(name types.NamespacedName, spec horizon.HorizonSpec) *horizon.Horizon {
```

Comment:
wondering if we can merge this into CreateHorizon following the interface we have in heat-operator.

Reply:
done